src/webAppCreator.ts

```diff
@@ -600,12 +647,36 @@ class WebsiteNameStep extends WebAppCreatorStepBase {
         return this._websiteName;
     }
 
-    get suggestedRGAndPlanName(): string {
-        return this._suggestedRGAndPlanName;
+    public suggestRelatedName(): Promise<string> {
```

Comment:
Can we rename this to `getSuggestedName`? The current name makes me think it will prompt the user to 'suggest' a name

Reply:
=> computeRelatedName